functional-samples/tutorial.custom-cursor/counter.js

```diff
@@ -0,0 +1,3 @@
+document.addEventListener('click', async () => {
```

Comment:
why is it async if you aren't using the return value?

Reply:
Good spot, thanks! That wasn't needed. Went ahead and removed it.